.gitignore

```diff
@@ -6,3 +6,6 @@ src_managed/
 project/boot/
 project/plugins/project/
 project/source.scala
+.bloop
```

Comment:
Super nit, but any reason not to just put these in your user/global ignore list instead of manually adding them to the repo itself?

Reply:
I always put them in repo gitignores due to work on projects with lots of committers who all have different user/global ignore lists. One less thing to deal with in PR reviews. :) I don't feel strongly about this though and I'm happy to remove the changes to .gitignore here.